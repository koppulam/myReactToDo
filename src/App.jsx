import React from 'react'
import ToDoList from './component/ToDoList.jsx'

class App extends React.Component {
	constructor() {
		super()
		console.log("test")
	}
	render() {
		return (
			<div>
				<ToDoList></ToDoList>
			</div>
			)
	}
}

export default App