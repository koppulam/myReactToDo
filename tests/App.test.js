// App.test.js 

import React from 'react';
import App from '../src/App';
import renderer from 'react-test-renderer';


describe("Testing App jsx using jest", function () {
	// body...
	it("Should generate snapshot and test", function() {
		const component = renderer.create(
	    <App >Facebook</App>
	  );
	  let tree = component.toJSON();
	  expect(tree).toMatchSnapshot();
	})
})
