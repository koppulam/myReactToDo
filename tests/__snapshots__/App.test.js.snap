// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App jsx using jest Should generate snapshot and test 1`] = `
<div>
  <div>
     list of toDos
    <ul>
      <li>
         
        <div>
          First toDoText
        </div>
         
      </li>
      <li>
         
        <div>
          second toDoText
        </div>
         
      </li>
      <li>
         
        <div>
          third toDoText
        </div>
         
      </li>
      <li>
         
        <div>
          fourth toDoText
        </div>
         
      </li>
      <li>
         
        <div>
          fifth toDoText
        </div>
         
      </li>
    </ul>
    <button
      onClick={undefined}
    >
       Add New To Do 
    </button>
  </div>
</div>
`;
